client/packages/system/src/Asset/AssetLogReasons/LogReasonCreateModal/LogReasonCreateModal.tsx

```diff
@@ -96,13 +94,8 @@ export const LogReasonCreateModal: FC<LogReasonCreateModalProps> = ({
           variant="ok"
           disabled={isInvalid}
           onClick={async () => {
-            // The default enum value 'NotInUse' will not be used, as it will fail the checkStatus validation first.
-            if (checkLogReasonStatus(draft.assetLogStatus))
```

Comment:
As stated, status is always defaulted to 'Functioning' in the seed, so have just displayed this and removed the status check